lib/utils/storage.ts

```diff
@@ -0,0 +1,61 @@
+import * as azure from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+const STORAGE_CONNECTION_STRING = process.env.AzureWebJobsStorage;
+
+const blobService = azure.createBlobService(STORAGE_CONNECTION_STRING);
+
+export function getBlobUrl(
+  containerName: string,
+  attachmentName: string
+): string {
```

Comment:
this is a good opportunity to use tagged types instead of plain strings (e.g. `DocumentDbAttachmentUrl`)